### PR TITLE
Include SteamSpacyTokenizer

### PR DESCRIPTION
**Pull request by my-master**

### CODE REVIEW

File: deeppavlov/configs/go_bot/gobot_dstc2.json [hunk 1]
```diff
@@ -7,13 +7,21 @@
     "name": "dialog_dataset"
   },
   "chainer": {
-    "in": ["x"],
-    "in_y": ["y"],
-    "out": ["y_predicted"],
+    "in": [
+      "x"
```
Comment (seliverstov) [inline deeppavlov/configs/go_bot/gobot_dstc2.json]: Мне кажется, что конфиг сильно раздувается и перестает умещаться на экране при таком способе написания. ИМХО он должен быть по возможности компактным.

Comment (my-master) [reply deeppavlov/configs/go_bot/gobot_dstc2.json]: All indents in my code are managed automatically by my IDE settings (PEP8, json formatting, etc.). Return the original formatting?

Comment (seliverstov) [reply deeppavlov/configs/go_bot/gobot_dstc2.json]: yes, please.

Comment (seliverstov) [reply deeppavlov/configs/go_bot/gobot_dstc2.json]: Merge new commits from 'dev' branch. It seems that tests fail due to incorrect fasttext version.

Comment (my-master) [reply deeppavlov/configs/go_bot/gobot_dstc2.json]: [Done](https://github.com/deepmipt/DeepPavlov/pull/104/commits/6b5ad319cf95320199243aa1f0f2436f729319e0). But beware that current formatting is manual and can't be maintained automatically.

Comment (my-master) [reply deeppavlov/configs/go_bot/gobot_dstc2.json]: [Done](https://github.com/deepmipt/DeepPavlov/pull/104/commits/e0a3c6155ff36b30d125860b05e42cbb48c101c6) merge dev.

Comment (my-master) [reply deeppavlov/configs/go_bot/gobot_dstc2.json]: Test passed.

**Pull request merged**